### ISSUE TITLE
UI: Minor visual tweaks

### ISSUE THREAD

**Issue by kevincrepin** [citizenos/citizenos-web#474]
Adding here some things I found in the new web.  Generally great work, just a couple small things to make it even better!

![Screenshot 2023-07-04 at 12 12 35](https://github.com/citizenos/citizenos-web/assets/26792281/af2bdd85-a4bb-4965-968d-b1150c77094c)
We should reduce the paragraph width here. The text lines become so long it makes it difficult to read, especially with the center alignment.

![Screenshot 2023-07-04 at 12 17 27](https://github.com/citizenos/citizenos-web/assets/26792281/b0ed96f6-0c96-4100-b6c3-c5d2fadc5a8f)
- Can the different features titles be bigger like in design?
- Normally you can open only one at a time and each section has its own illustrations: https://www.figma.com/file/eHBSunqUv10fMcWmiEEXiv/Website?type=design&node-id=1429%3A8982&mode=design&t=fDCDAcZt4ZwhPTYP-1

![Screenshot 2023-07-04 at 12 20 28](https://github.com/citizenos/citizenos-web/assets/26792281/13895333-d66c-4179-bf84-95cd77e62f2b)
- The button here has a drop shadow but shouldn't have
- The button hover border should be also in yellow (cf https://www.figma.com/file/eHBSunqUv10fMcWmiEEXiv/Website?type=design&node-id=1268%3A6616&mode=design&t=fDCDAcZt4ZwhPTYP-1)
- Generally the (paragraph) width here is bit too much. I kept it narrower in design so it's more readable. You can compare with design and see which is more comfortable to read: https://www.figma.com/file/eHBSunqUv10fMcWmiEEXiv/Website?type=design&node-id=2580%3A12182&mode=design&t=fDCDAcZt4ZwhPTYP-1

![Screenshot 2023-07-04 at 12 23 11](https://github.com/citizenos/citizenos-web/assets/26792281/d06eb65b-802d-47dd-aedf-96dfb68d27dc)
The text sizes here are bit too big. 14px is used for this "signature" bio.

![Screenshot 2023-07-04 at 12 24 27](https://github.com/citizenos/citizenos-web/assets/26792281/1e3ca6ec-f1e6-4546-b276-57435ec90383)
The arrow should be same colour as the link + the alignment also doesn't seem (visually) in the middle.

![Screenshot 2023-07-04 at 12 28 29](https://github.com/citizenos/citizenos-web/assets/26792281/3a80a5b6-a628-49f0-9fac-2ee26996f231)

- on mobile this should be a dropdown (cf https://www.figma.com/file/eHBSunqUv10fMcWmiEEXiv/Website?type=design&node-id=2582%3A13501&mode=design&t=fDCDAcZt4ZwhPTYP-1)

- General comment for mobile: Seems like the margins and paddings aren't entirely correct. Can you pls check again compared to design?

**Comment by oksks** [comment]
@devosama1 can you pls address this when you have time?

**Comment by devosama1** [comment]
@oksks This is resolved all, except one which says "main platform features." Working on that atm, but rest can be tested

**Comment by oksks** [comment]
@devosama1 the "main platform features" block's font size is still awaiting action?

**Comment by devosama1** [comment]
@oksks yes. Ill redo this block after other priority items. There isn't an option in elementor that allows me to customize it to an extent that works for the design at the moment. So ill custom code it and inject as a shortcode.

**Comment by devosama1** [comment]
![image](https://github.com/citizenos/citizenos-web/assets/56334857/a59f783b-c77d-438a-b8b6-cb108f772cba)
https://citizenos.com/?page_id=6370 

Please test the transition, the images are changing on clicking the accordion titles. They are changing as expected now. All finished with this task list

**Comment by oksks** [comment]
Looks great, I love it! But could you please force the other sections collapse, when one is active? @devosama1
